common/presenters/event-to-timeline-item-component.js

```diff
@@ -58,6 +62,7 @@ const eventToTimelineItemComponent = (moveEvent, move) => {
       heading,
       description,
       timestamp,
+      createdBy,
```

Comment:
If you look at line 55 I think Alex set `byline` to an empty string with a comment that it can be substituted out once we have this information.

I don't think you'll need the translation though, as the component already supports the addition of "by" so could just be:

```javascript
return {
  id,
  ...getItem({
    ...
    byline: createdBy,
    ...
  }),
}
```

Reply:
yes, that's so much easier... done!